.github/workflows/purge-test-resources.yaml

```diff
@@ -93,5 +93,17 @@ jobs:
           cat ${{ env.AZURE_RG_DELETE_LIST_FILE}} | while read line
           do
               echo " * $line" >> $GITHUB_STEP_SUMMARY
-              az group delete --resource-group $line --yes --verbose
+              az group delete --resource-group $line --yes --verbose --no-wait
```

Comment:
I think that because we have a lot of resource groups, it takes more than 5 minutes to delete them. Within that 5 minutes, the session loses its access because login is only granted for 5 minutes. Please see: https://github.com/Azure/login/issues/372.

I may be wrong about this. Open to suggestions. cc/ @youngbupark

Reply:
I think this is a similar issue to this one: https://github.com/radius-project/radius/issues/7494. There was a regression in the az cli with the login expiration so they provided some workarounds here: https://github.com/Azure/azure-cli/issues/28737#issuecomment-2051026300

But we already use a service principal and that still seems to be failing for us. I think they plan to release a fix on 4/30 so we may need to revisit this once that version is out

Reply:
Looks like az login issues 5 mins expiry token. I am ok to add `--no-wait`. The fix will be released in 2.60 CLI.